Controller/ActionController.php

```diff
@@ -66,13 +66,11 @@ public function processAction(Request $request)
         $encryptorFactory->setProjectId($tokenInfo["owner"]["id"]);
         $encryptorFactory->setStackId(parse_url($this->container->getParameter('storage_api.url'), PHP_URL_HOST));
         $configData = isset($requestJsonData["configData"]) ? $requestJsonData["configData"] : [];
-        if (in_array("encrypt", $component["flags"])) {
-            try {
-                $configData = $encryptorFactory->getEncryptor()->encrypt($configData);
-                $configData = $encryptorFactory->getEncryptor()->decrypt($configData);
-            } catch (\Keboola\ObjectEncryptor\Exception\UserException $e) {
-                throw new UserException($e->getMessage(), $e);
-            }
+        try {
+            $configData = $encryptorFactory->getEncryptor()->encrypt($configData);
```

Comment:
nešlo by to `encrypt+decrypt` zúžit pouze na `decrypt`?

Reply:
myslim, ze sme to tam pridavali zrovna kvoli nejakemu bugu.

Reply:
jj - https://github.com/keboola/docker-bundle/commit/1f955d87b72e10cd69716cff069d8c38e083b9bf
ale už to třeba není potřeba?

Reply:
myslim, ze je ... co sync akcie typu test credentials? tam sa to stale posiela z UI nezasifrovane

Reply:
třeba to nová encryption libka zchroustá :-)

Reply:
nezchrousta, protoze ma fallback na tu starou. V principu by to zchroustat slo, tzn. misto vyjimky "is not an encrypted value" tam nechat tu hodnotu jak je, Jenze, protoze je tam fallback na mcrypt a ten je schopnej to rozsfirovat i kdyz to neni sifra, takze by se tam pak objevila kravina

Reply:
jasně, mcrypt.